apps/web/pages/index.vue

```diff
@@ -136,10 +122,24 @@
 import { SfButton } from '@storefront-ui/vue';
 const viewport = useViewport();
 const { t } = useI18n();
-definePageMeta({ pageType: 'static' });
-
 const { data: categoryTree } = useCategoryTree();
 const recommendedProductsCategoryId = ref('');
+definePageMeta({ pageType: 'static' });
+
+useHead({
+  link: [
+    {
+      rel: 'preload',
+      href: viewport.isLessThan('md') ? '/images/homepage-hero-bg-mobile.avif' : '/images/homepage-hero-bg.avif',
```

Comment:
I dont think thats accurate, as we have a different Images structure now:

![Screenshot 2024-05-22 at 18 03 28](https://github.com/plentymarkets/plentyshop-pwa/assets/149154151/33542ead-175d-4a64-bd56-08a5417d8ca0)